File: Models/ProjectSkill.cs
namespace MostaqlK.Models;

/// <summary>
/// A single skill tag associated with a project (e.g. "PHP", "تصميم شعارات").
/// </summary>
public sealed class ProjectSkill
{
    public string Name { get; set; } = string.Empty;

    public string? Url { get; set; }
}

File: .repertoire/python/src/mostaql/models/project_skill.py
"""A single skill tag associated with a project (C# Models/ProjectSkill)."""

from dataclasses import dataclass

__all__ = ["ProjectSkill"]


@dataclass(frozen=True, slots=True)
class ProjectSkill:
    """A single skill tag associated with a project (e.g. "PHP", "تصميم شعارات")."""

    name: str
    url: str | None = None
